KarhooUISDK/Extensions/KarhooSDKExtensions/Quote+Extensions.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Quote+Extensions.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 11/01/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+extension Quote {
+
+    func setExpirationDate(using validity: Int) {
+        QuoteDatesHelper.setExpirationDate(
+            of: self,
+            date: Date().addingTimeInterval(TimeInterval(validity))
+        )
+    }
+
+    var quoteExpirationDate: Date? {
+        QuoteDatesHelper.getExpirationDate(of: self)
+    }
+
+}
+                                           
+enum QuoteDatesHelper {
```

Comment:
Why `enum`? It doesn't describe the various states of something.

Reply:
Enum is the type that can't have any instance. Therefore, it can be used as replacement of (not presence in Swift, unfortunately), namespace-syntax.

Reply:
Your intention is clear now, but I think it's kind of not that intuitive across language backgrounds at first glance. Having a class or a struct with static methods would be clearer as a convention (for Android devs doing code reviews for example). Your solution is perfectly valid, I'm just worried about keeping standards that are easy to understand across the whole team.

Reply:
No problem - for sake of clarity, I can change type to struct. Every company and project has it's an own way of doing things :)